src/components/InteriorLeftNav/InteriorLeftNav-test.js

```diff
@@ -46,6 +46,22 @@ describe('InteriorLeftNav', () => {
           .matchesElement(<div data-child-div>foo</div>)
       ).toBe(true);
     });
+
+    it('renders a interior left nav with initialOpenState to be true', () => {
```

Comment:
can you change the test text to reflect the prop name change from `initialOpenState` to `open`?